package.json

```diff
@@ -7,7 +7,7 @@
   "jsnext:main": "dist/formidable-oss-badges.es.js",
   "module": "dist/formidable-oss-badges.es.js",
   "dependencies": {
-    "buble": "0.19.6"
+    "styled-components": "4.1.1"
```

Comment:
This should probably not be pinned and be a peer and dev dependency (where peer allows for v5, so `>= 4.0.0`)